src/main/java/org/stuartgunter/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -463,4 +494,15 @@ public Cluster build(MetricRegistry metrics, HealthCheckRegistry healthChecks) {
 
         return cluster;
     }
+
+    private void validateDnsContactPointsType() {
+        if (DNS.equals(contactPointsType) && contactPoints.length > 1) {
+            throw new ValidationException("Cannot specify more than one contact point when contactPointsType is DNS.");
+        }
+    }
+
+    public enum ContactPointsType {
+        IP,
```

Comment:
I'm not convinced about the name `IP`, as it suggests an actual IP address is required when a DNS name would do fine.

I'm wondering whether we should abandon the idea of the `contactPointsType` property and just call `addContactPoints(String)` for each item in `contactPoints`. The only scenario where that would be a problem (I think) is when people are using DNS entries with multiple A records, but hoping to only use one of them - which seems like a strange thing to do.

I just worry this ends up being more complicated to use.